pkgs/development/web/flyctl/default.nix

```diff
@@ -43,6 +43,8 @@ buildGoModule rec {
       --fish <($out/bin/flyctl completion fish) \
       --zsh <($out/bin/flyctl completion zsh)
     ln -s $out/bin/flyctl $out/bin/fly
+    # If autoupdate is true, we get into an autoupdate loop
+    $out/bin/flyctl settings autoupdate disable
```

Comment:
This is written to the user directory. Specifically `~/.fly/config.yml` is being updated with `auto_update: false`. This is unlikely to change anything for the installed / deployed part.
If we want to disable the autoupdate because it doesn't work correctly with nixpkgs, we may need to remove it in the code.

Reply:
We could change the default for example: https://github.com/superfly/flyctl/blob/e1eef70774c9b1601a68d88713816053528e5d73/internal/config/config.go#L144

Reply:
@viraptor I've never patched code with a nixpkg before - I'll take at any existing examples, but in this case would it be easier to generate and apply a patch file or just `sed` (or similar) the specific line?

Reply:
A patch is likely more effective here... It's a larger file and may include a few cases of a similar line. You can make a patch and add it to `patches = [ ... ];` list in the package.

Although you could also ask upstream to provide some compile-time or environment-variable managed way to turn it off.